### DISCUSSION TITLE
ALB -> Lambda with LWA Streaming?

### DISCUSSION THREAD

**Discussion by bestickley** [awslabs/aws-lambda-web-adapter#464] [Q&A] [answered]
Hi, I'm building a solution in AWS GovCloud so I cannot use CloudFront or Function URLs (as of May 23rd 2024), so I'm looking to other strategies to stream from Lambda to a client. I've investigated ALB but based on the [docs](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/lambda-functions.html), I don't think I can. I wanted to ask the question here though in case I'm (hopefully) wrong.

**Comment by bnusunny** [comment, marked as answer]
No, ALB to Lambda does not support response streaming. 

You could build a proxy layer on Fargate or App Runner to invoke Lambda functions using invokeWithReponseStream API call.
